run-create/README.md

```diff
@@ -0,0 +1,66 @@
+# Create Test Run
+
+This action creates a new test run in Qase TMS and returns its ID.
+
+## Inputs
+
+### token
+
+**Required**. Qase API token. You can get it in your account settings.
+
+### project
+
+**Required**. Project code where the test run will be created.
+
+### title
+
+**Required**. Title of the test run.
+
+### description
+
+Description of the test run.
+
+### environment
+
+Environment of the test run.
+
+### milestone
+
+Milestone of the test run.
+
+### plan
+
+Plan of the test run.
+
+### verbose
+
+Verbose mode. Default: false.
+
+## Outputs
+
+### id
+
+ID of the created test run.
+
+## Example usage
+
+```yaml
+steps:
+  - uses: actions/checkout@v3
+
+  - uses: qase-tms/gh-actions/run-create
+    id: create
+    with:
+      token: ${{ secrets.QASE_TOKEN }}
+      project: PRJ_CODE
+      title: Test Run Title
+      description: Test Run Description
+      environment: Production
+      milestone: Release 1.0
+      plan: Regress
```

Comment:
Same with plans, I don't see it in the API. Unless it's done by the CLI app.

https://developers.qase.io/reference/create-run

Reply:
Fixed